src/services/OswServiceTaskBLEServer.cpp

```diff
@@ -301,25 +301,19 @@ void OswServiceTaskBLEServer::BatteryLevelStatusCharacteristicCallbacks::onRead(
 
 void OswServiceTaskBLEServer::CurrentTimeCharacteristicCallbacks::onRead(NimBLECharacteristic* pCharacteristic) {
     time_t offset = 0;
```

Comment:
?

Reply:
why are you use offset and init to `0`.

Reply:
Because we have no function called `getUTCDate` - we have only functions for local dates, or the UTC time. Hence, I needed to provide the offset for this. But you are right, I've changed the time getter to match the pattern too.